ParcelInvoiceMaker/ParcelInvoiceMaker/Model/Discount.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Discount.swift
+//  ParcelInvoiceMaker
+//
+//  Created by Hong yujin on 4/12/24.
+//
+
+import Foundation
+
+enum Discount: Int {
+    case none, vip, coupon
+    var strategy: DiscountStrategy {
+        switch self {
+        case .none:
+            NoneStrategy()
+        case .vip:
+            VipStrategy()
+        case .coupon:
+            CouponStrategy()
+        }
+    }
+}
+
+protocol DiscountStrategy {
+    func applyDiscount(deliveryCost: Int) -> Int
```

Comment:
요것도 취향의 영역인데요,
저는 영어 한 문장이 자연스럽게 읽히는 것을 좋아해서 아규먼트를 많이 활용하는 편입니다!

```suggestion
    func applyDiscount(with deliveryCost: Int) -> Int
```